src/main/kotlin/no/nav/syfo/domain/Virksomhetsnummer.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.syfo.domain
+
+data class Virksomhetsnummer(val value: String) {
+    private val nineDigits = Regex("\\d{9}")
+
+    init {
+        if (!nineDigits.matches(value)) {
+            throw IllegalArgumentException("$value is not a valid Virksomhetsnummer")
+        }
```

Comment:
Obs: Regexen vil vel også matche om det er flere enn 9 siffer?

Reply:
Er ikke {9} lik eksakt 9?

Reply:
Fikset regex. Endret også for AktorId og Fodselsnummer.